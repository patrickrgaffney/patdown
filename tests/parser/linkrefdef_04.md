[foo]: /url '
title
line1
line2
'