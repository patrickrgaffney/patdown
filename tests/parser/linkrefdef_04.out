LINK_REFERENCE_DEF: [foo]: /url '
title
line1
line2
'